packages/lesswrong/components/form-components/FormComponentDateTime.tsx

```diff
@@ -40,7 +52,6 @@ const styles = (theme: ThemeType): JssStyles => ({
       background: theme.palette.panelBackground.default,
       boxShadow: theme.palette.boxShadow.moreFocused,
       border: `1px solid ${theme.palette.grey[55]}`,
-      bottom: 30,
```

Comment:
We had this to make the date picker appear above the input. By default it appears below, and in this case we actually want it to appear below because the dates are at the top of the page. So we pulled this out into a separate class above.